knowledge_base/dashboard_nyc_taxi/resources/nyc_taxi_trip_analysis.job.yml

```diff
@@ -0,0 +1,19 @@
+resources:
+  jobs:
+    my_project_job:
+      name: my_project_job
```

Comment:
Please name this something more descriptive, for example "Daily NYC Taxi Trip Analysis refresh".